src/main/java/org/opensrp/web/utils/MultimediaUtil.java

```diff
@@ -0,0 +1,14 @@
+package org.opensrp.web.utils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MultimediaUtil {
+
+	public static Boolean hasSpecialCharacters(String fileName) {
+		Pattern special = Pattern.compile("[:\"\\\\?|/'<>*/\r\t\f\n\0]");
```

Comment:
Compiling patterns is a bit expensive. Move the pattern to a static variable on this class so that its not always run

Reply:
`\\\\` is repeated shouldn't this be `\\`

Also Zero '0' is not a special character but `\0` is . The current regex matches `0` alone.

A single quotation mark "'" should not be considered a special character. It is used to represent files with spaces also `\ ` should be a valid pattern as illustrated below

`individual\ issue\ 1.png` and `'individual issue 1.png'` should be considered valid filenames

Reply:
As per my understanding, following special characters won't be allowed in fileName or entity Id depending on the end-point.
**" ? | < > * / \r \t \n \f \0**

Reply:
@githengi Updated PR according to the feedback. Please review. Thanks!